credsweeper/filters/value_ip_check.py

```diff
@@ -32,9 +32,10 @@ def run(self, line_data: LineData, target: AnalysisTarget) -> bool:
         with contextlib.suppress(Exception):
             ip = ipaddress.ip_address(line_data.value)
             if 4 == ip.version:
-                line_lower = target.line.lower()
+                # use line_strip_lower due the property should be cached already
+                line_strip_lower = target.line_strip_lower
                 for i in ValueIPCheck.FALSE_POSITIVE_MARKERS:
-                    if i in line_lower:
+                    if i in line_strip_lower:
```

Comment:
Is it okay to use `target.line_strip` instead of `target.line`?

Reply:
Yes, it is OK. 
Shorter string - quickly process. Whitespaces are not necessary to research

Reply:
I see. :)